### ISSUE TITLE
player storage system

### ISSUE THREAD

**Issue by Blumlaut** [Blumlaut/EasyAdmin#262]
~~it'd probably be feasable to move all the easyadmin-generated files into its own folder (except for the permissions file), although that might be too big of a task.~~

~~will keep this one on the backburner, not a focus.~~

A User Storage system for saving warnings, kicks and whatnot might be possible if we store their last time they were on the server as well, and cull old players which have not returned, the only issue this would pose in my opinion would be database size, so the culling might need to be quite aggressive (<30 days?) 

Related/samey to #469, might just close that issue in favour of this, i reckon actions on specific players are more important than _just_ those actions themselves

**Comment by Blumlaut** [comment]
> Just save information to a mysql database like every other administration system does.

No. EasyAdmin's whole point is that it does not have external dependencies.

**Comment by mcNuggets1** [comment]
No serious server writes data to disk. It's just stupid by design to not use the main way people tend to store data, which is via oxmysql/mysql-async (for years).
Also you could support both which would drastically improve EasyAdmin's performance and usability.

**Comment by Blumlaut** [comment]
>  Also you could support both which would drastically improve EasyAdmin's performance and usability.

Great! Are you up to maintain that system in case syntax changes and provide a proper fully-automatic migration system? no? too bad!
